src/deadline/client/api/_loginout.py

```diff
@@ -58,7 +58,8 @@ def _login_deadline_cloud_monitor(
             time.sleep(0.5)
     except FileNotFoundError:
         raise DeadlineOperationError(
-            f"Could not find Deadline Cloud Monitor at {deadline_cloud_monitor_path}. Please ensure Deadline Cloud Monitor is installed correctly and setup the {profile_name} profile again."
+            f"Could not find Deadline Cloud Monitor at {deadline_cloud_monitor_path}."
+            f"Please ensure Deadline Cloud Monitor is installed correctly and set up the {profile_name} profile again."
```

Comment:
Based on the branding guidelines, `monitor` should actually be lower-cased here while Deadline Cloud remains capitalized. Might be worth updating more than just here for that.

Reply:
Actually that was one of my questions, because here we were using `M`onitor unlike other places. Will update it.